_includes/platform-support.html

```diff
@@ -0,0 +1,37 @@
+<div class=".container-fluid mb-4">
+    <div class="full-bleed">
+        <h2 class="h3 mb-3 fw-bold text-stride">Develop once, deploy cross-platform</h2>
+        <div class="grid">
+            <div class="main">
+                <img alt="stride-logo" height="37" width="124" src="/images/svg/stride-logo-dark.svg" />
+            </div> 
+
+            <div class="main center">Windows <i class="fa-brands fa-windows"></i></div>
+            <div class="main center">MacOS <i class="fa-brands fa-apple"></i></div>
+            <div class="main center">Linux <i class="fa-brands fa-linux"></i></div>
+            <div class="main center">Android <i class="fa-brands fa-android"></i></div>
+            <div class="main center">IOS <i class="fa-brands fa-apple"></i></div>
+
+            <div class="main">Deploy</div>
+            <div class="secondary check"><i class="fa-solid fa-circle-check"></i></div>
+            <div class="secondary check"><i class="fa-solid fa-circle-check"></i></div>
+            <div class="secondary check"><i class="fa-solid fa-circle-check"></i></div>
+            <div class="secondary check"><i class="fa-solid fa-circle-check"></i></div>
+            <div class="secondary check"><i class="fa-solid fa-circle-check"></i></div>
+
+            <div class="main">Code only</div>
```

Comment:
I would replace word "**Code only**" with word "**Runtime**".
That's because Runtime is a wider term and AFAIK code-only is a part of Stride Community toolkit and It's basically one of two flavors of developing game. First is unity-like (default one) in which you visually adding or creating an assets by using GameStudio. This however does not mean that you need GameStudio to run your game.

Reply:
Maybe good idea would be adding Tooltip that this also includes code only approach (example: https://fluent2.microsoft.design/components/web/react/tooltip/usage/)

Reply:
I agree that "Code Only" isn't the best term for this, though it is what has been used so far for it. "Runtime" might not be the best either as imo it has more of a connotation of the actual running application (especially true if you look at Unity). Maybe "Framework" would make sense? Generally I see the term used when refering to 'game engines' that don't have a visual editor. So, just the API/code.

Reply:
Now you see, why this PR might take some time to resolve 🤣🙈